lib/rebuys.post.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Handles rebuying after a sell by setting up limit orders on the books using special config vars
+ * Sample env config:
+// should the engine only create and manage the limit orders and not make normal accumulation trades
+// useful for testing this feature
+CPBB_REBUY_ONLY: false,
+// maximum dollar value consumed by limit order placements
+CPBB_REBUY_MAX: 50,
+// rebuy logic will place up to  orders at this size until CPBB_REBUY_MAX is reached
+CPBB_REBUY_SIZE: ".0001,.0001,.0002,.0002,.0003,.0003,.0004,.0004,.0005,.0005",
+// rebuy at these percentage drop targets (-1%, -2%, etc)
+// note: you have to define at least the number of points in CPBB_REBUY_SIZE
+CPBB_REBUY_AT: "-1,-2,-4,-8,-16,-20,-30,-40,-50,-80",
+ */
+const { add, divide, format, multiply } = require('mathjs');
+const config = require('../config');
+const fs = require('fs');
+const log = require("./log");
+const memory = require("../data/memory");
+const processOrder = require('./process.order');
+module.exports = async (basePrice) => {
+  // starting percentage down from basePrice to place limit orders
+  const dropPoints = process.env.CPBB_REBUY_AT.split(',').map(p=>divide(Number(p), 100));
```

Comment:
what if we consume the rebuy configs in the config module and validate it at startup? There are a few misconfiguration options potential here. If someone defines the SIZE with fewer options than AT for instance.